src/app/App.blocks.test.tsx

```diff
@@ -111,6 +113,22 @@ describe("App Router", () => {
 	});
 });
 
+const renderComponent = (path = "/", options = {}) => {
+	render(
+		<ErrorBoundary FallbackComponent={ApplicationError}>
```

Comment:
There were some offended tests with `[Error: ErrorBoundaryContext not found]` error, that is why wrapped the component with the error boundary component. It was happening due to `const { showBoundary } = useErrorBoundary();` call